### PR TITLE
Replace old memdb with level-mem

### PR DESCRIPTION
**Pull request by vweevers**
Same API, but up to date.

### CODE REVIEW

File: test/hooks.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ tape('[promises] test level-hookdown', async function (t) {
 })
 
 tape('test level-hookdown series', function (t) {
-  var level = MemDB({ valueEncoding: 'json' })
+  var level = mem(null, { valueEncoding: 'json' })
```
Comment (vweevers) [inline test/hooks.js]: This difference between `memdb` and `level-mem` will be fixed by https://github.com/Level/packager/pull/95

**Comment by vweevers** [comment]
Build failure is due to Travis having an outdated npm cache.

**Comment by vweevers** [comment]
@bcomnes I forgot to mention, `subleveldown` will likely stop working with `memdb`. In case you're using this combination of modules anywhere else.

**Comment by bcomnes** [comment]
@vweevers Is there anything I can read about that to understand the issue and potential solutions?

**Comment by vweevers** [comment]
https://github.com/Level/subleveldown/issues/83

**Comment by bcomnes** [comment]
Thanks will check that a out

**Comment by bcomnes** [comment]
Apologies, I read this in reverse, that level-mem is dropping support for subleveldown or something.  Everything looks great. Thanks!

**Pull request merged**